src/main/java/cz/codecamp/logger/formaters/DummyFormatter.java

```diff
@@ -0,0 +1,11 @@
+package cz.codecamp.logger.formaters;
+
+import cz.codecamp.logger.FormatterInterface;
+import cz.codecamp.logger.Message;
+
+public class DummyFormatter implements FormatterInterface {
```

Comment:
Pokud se `DummyFormatter` používá pouze v testech, není potřeba ho mít v `src/main` - stačí ho mít v `src/test`... Má to taky výhodu, že pro lepší code coverage na něj není potřeba psát test :)